tee-worker/scripts/litentry/release/prepare.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+set -euo pipefail
+
+# Edit the following variable(s):
+WORKER_DIR=/tmp/worker/
+
+RUNNING_MODE_CONFIG=$1/config.json
+PRIVATE_ACCOUNT_JSON=$1/private_account.json
+INTEL_KEY=$1/key_production.txt
+INTEL_SPID=$1/spid_production.txt
+
+##############################################################################
+# Don't edit anything from here
+if [ "$#" -ne 1 ]; then
+  echo "Error: Please provide one and only one secret path as input."
+  exit 1
+fi
+if [[ ! -e "$RUNNING_MODE_CONFIG" ]]; then
+  echo "Error: $RUNNING_MODE_CONFIG is not a valid path."
+  exit 1
+fi
+if [[ ! -e "$PRIVATE_ACCOUNT_JSON" ]]; then
+  echo "Error: $PRIVATE_ACCOUNT_JSON is not a valid path."
+  exit 1
+fi
+if [[ ! -e "$INTEL_KEY" ]]; then
+  echo "Error: $INTEL_KEY_PATH is not a valid path."
+  exit 1
+fi
+if [[ ! -e "$INTEL_SPID" ]]; then
+  echo "Error: $INTEL_SPID_PATH is not a valid path."
+  exit 1
+fi
+
+if [[ ! -e "$WORKER_DIR" ]]; then
+  mkdir -p $WORKER_DIR
+fi
+
+# Generate keys and copy around.
+SRC_DIR=$(dirname "$0")
+cd $SRC_DIR
+
+./integritee-service signing-key | grep -oP '^Enclave account: \K.*$$' > enclave_account.txt
```

Comment:
It seems I don't find where this binary is from

Reply:
It's from generated package.